EosGround/radioDriver.py

```diff
@@ -30,7 +34,7 @@
 print("connected to database")
 
 # sets up digi
-PORT = "/dev/cu.usbserial-FT5PFML62"
+PORT = "COM6"
```

Comment:
i feel like this should be a param, everyone changes this every time for whatever their local setup is